recipe/meta.yaml

```diff
@@ -7,17 +7,18 @@ package:
 source:
   fn: cmt-v{{ version }}.tar.gz
   url: https://github.com/lucastheis/cmt/archive/v{{ version }}.tar.gz
-  sha1: 7482caefa5359a351ac834920a092b5594098d70
+  sha256: 252d1029c62889a533d50e017445407e1de653cac8bee48ecdc34e4d8fd6e7ce
   patches:
-    # Workaround to ensure `gcc` compiler is used on Mac.
-    # See issue ( https://github.com/lucastheis/cmt/issues/10 ).
-    # No patch submitted as this will require discussion with the developer.
+    # Patch to avoid passing `-stdlib` to the compiler generally
+    # See issue ( https://github.com/lucastheis/cmt/issues/25 ).
+    # Patch for this has been submitted.
+    # See pull request ( https://github.com/lucastheis/cmt/pull/27 ).
     - setup.py.patch
```

Comment:
Opened issue ( https://github.com/lucastheis/cmt/issues/25 ) about the problem and PR ( https://github.com/lucastheis/cmt/pull/27 ) with a proposed fix. This is only build system related and effects are ability to use `gcc`.